src/main/resources/application.yml

```diff
@@ -1,7 +1,6 @@
 
 spring:
-  datasource:
-    url: jdbc:h2:mem:livecodingturkey
-    driver-class-name: org.h2.Driver
-  h2:
-    console.enabled: true
\ No newline at end of file
+  data:
+    mongodb:
+      uri: mongodb+srv://furkan:furkan@cluster0.n83xn.mongodb.net/?retryWrites=true&w=
```

Comment:
`uri: ${MONGO_URI}` and add an environment variable to your app like `MONGO_URI=mongodb+srv.....`